usr.sbin/freebsd-update/freebsd-update.sh

```diff
@@ -2200,6 +2210,13 @@ fetch_run () {
 	done
 	fetch_tagsanity || return 1
 
+	# If there are no updates upstream there is no need for
+	# further processing (except forced fetch).
+	if [ $FORCEFETCH -eq 0 ]; then
+		fetch_compare_new_tag_to_current_tag &&
+		  echo "No update required." && return 0
```

Comment:
style: indent with 4, not 2 spaces.
Sorry I missed this one the first time, style corrections are manual at the moment. :disappointed: